SF_Users_API/db/migrations/20171030225656_initial.js

```diff
@@ -0,0 +1,33 @@
+
+exports.up = function(knex, Promise) {
+  return Promise.all([
+    knex.schema.createTable('sf_users', function(table) {
+      table.increments('id').primary();
+      table.string('username');
```

Comment:
I think we need to add another column here for the OAuth id we get for tracking purposes.